docs/articles_en/openvino_workflow/gen_ai.md

```diff
@@ -0,0 +1,165 @@
+
+The table below summarizes differences of both Hugging Face and Native approaches. 
+
+.. list-table::
+   :widths: 20 25 55
+   :header-rows: 1
+
+   * -  
+     - Hugging Face through OpenVINO
+     - OpenVINO Native API
+   * - Model support
+     - Wide set of Models
+     - Wide set of Models
+   * - APIs
+     - Python (Hugging Face API)
+     - Python, C++ (OpenVINO API)
+   * - Model Format
+     - Source Framework / OpenVINO
+     - OpenVINO
+   * - Inference code
+     - Hugging Face based
+     - Custom inference pipelines
+   * - Additional dependencies
+     - As much as Hugging Face has
+     - Ligthweight (e.g. numpy, boost, etc.)
+   * - Application footprint
+     - Large
+     - Small
+   * - Pre/post-processing and glue code
+     - Available at Hugging Face OOB
+     - OpenVINO samples and notebooks
+   * - Performance
+     - Good
+     - Best
+
+
+Running Generative AI models using Hugging Face Optimium-Intel 
+##############################################################
+
+Prerequisites
++++++++++++++++++++++++++++
+
+* Create a Python environment
+* Install Optimium-Intel running:
+
+.. code-block:: console
+
+    pip install optimum[openvino,nncf]
+
+
+To start using OpenVINO as a backend for Hugging Face you should change original Hugging Face code in two places. 
+
+.. code-block:: diff
+
+    -from transformers import AutoModelForCausalLM
+    +from optimum.intel. import OVModelForCausalLM
+
+    model_id = "meta-llama/Llama-2-7b-chat-hf"
+    -model = AutoModelForCausalLM.from_pretrained(model_id)
+    +model = OVModelForCausalLM.from_pretrained(model_id, export=True)
+
+
+After that, you can call ``.save_pretrained()`` method to save model to the folder in the OpenVINO Intermediate Representation and use it further.
+
+.. code-block:: python
+
+    model.save_pretrained(model_dir)
+
+
+Alternatively, you can download and convert model using CLI interface: ``optimum-cli export openvino --model meta-llama/Llama-2-7b-chat-hf llama_openvino``
+In this case, you can load the converted model in OpenVINO representation directly from the disk:
+
+.. code-block:: python
+
+    model_id = "llama_openvino"
+    model = OVModelForCausalLM.from_pretrained(model_id)
+
+
+By default, inference will run on CPU. To select a different inference device, for example GPU, add `device="GPU"` to the `.from_pretrained()` call. To switch to a different device after the model has been loaded, use the `.to()` method. The device naming convention is the same as in OpenVINO native API:
+
+.. code-block:: python
+
+    model.to("GPU")
+
+
+Optimum-Intel API also provides out-of-the-box model optimization through weights compression using NNCF which substantially reduces the model footdpring and inference latency:
```

Comment:
not sure but I checked that Optimum Intel is spelled as two words in other docs, correct me if that's wrong